packages/@aws-cdk/aws-applicationautoscaling/lib/scalable-target.ts

```diff
@@ -225,6 +226,14 @@ export interface ScalingSchedule {
    * @default No new maximum capacity
    */
   readonly maxCapacity?: number;
+
+  /**
+   * The time zone used when referring to the date and time of a scheduled action,
+   * when the scheduled action uses an at or cron expression.
+   *
+   * @default - No specific time zone.
+   */
+  readonly timezone?: string;
```

Comment:
I don't necessarily love typing this as a string, because I think we can do better, but something similar was already introduced in #17330. Can we at least have the documentation explicitly state what kind of string we're looking for here, like we do in #17330?